cc/tf_bt_utils.cc

```diff
@@ -127,6 +128,44 @@ void WriteGameExamples(const std::string& gcp_project_name,
             << examples.size() << std::endl;
 }
 
+void WriteEvalRecord(const std::string& gcp_project_name,
+                     const std::string& instance_name,
+                     const std::string& table_name,
+                     const MctsPlayer& player,
+                     const std::string& black_player_name,
+                     const std::string& white_player_name) {
+  Table table(CreateDefaultDataClient(gcp_project_name, instance_name,
+                                      google::cloud::bigtable::ClientOptions()),
+              table_name);
+
+  // Retrieve the game counter from the Bigtable and increment it atomically.
+  using namespace google::cloud::bigtable;
+  auto rule =
+      ReadModifyWriteRule::IncrementAmount("metadata", "game_counter", 1);
+  auto row = table.ReadModifyWriteRow("table_state", rule);
+  uint64_t game_counter = 0;
+  // TODO assert size == 1?
```

Comment:
I was talking about the size of row.cells(),
e.g. can I add `assert(row.cells().size() == 1);`

Reply:
The `table_state` row may contain more than one cell, especially in the future. However, the documentation says that it returns "A Row containing the new contents of all cells modified by the request," so if you have only updated one cell in the row, it may be safe to insist on its size.

Reply:
Okay, I'll investigate in the future and leave this as a better formatted todo today.